From 397a8fd06bd576cae7f1fae71dbfd9fbce30dc66 Mon Sep 17 00:00:00 2001
From: K900 <me@0upti.me>
Date: Fri, 17 Nov 2023 07:48:13 +0300
Subject: [PATCH] python2/mk-python-derivation: disable catchConflictsHook

It's broken on Py2. Hopefully people still using 2 know what they're doing?
---
 .../interpreters/python/python2/mk-python-derivation.nix       | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pkgs/development/interpreters/python/python2/mk-python-derivation.nix b/pkgs/development/interpreters/python/python2/mk-python-derivation.nix
index 1a6f9c784cf675..6d51b51b215d43 100644
--- a/pkgs/development/interpreters/python/python2/mk-python-derivation.nix
+++ b/pkgs/development/interpreters/python/python2/mk-python-derivation.nix
@@ -57,7 +57,8 @@
 # Raise an error if two packages are installed with the same name
 # TODO: For cross we probably need a different PYTHONPATH, or not
 # add the runtime deps until after buildPhase.
-, catchConflicts ? (python.stdenv.hostPlatform == python.stdenv.buildPlatform)
+# FIXME: disabled for Python 2 because broken
+, catchConflicts ? false

 # Additional arguments to pass to the makeWrapper function, which wraps
 # generated binaries.
